### PR TITLE
finished courses cubit test

### PR DESCRIPTION
**Pull request by aleke88884**

### CODE REVIEW

File: test/features/courses/ui/courses_cubit_test.dart [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+void main() {
+  late CoursesCubit coursesCubit;
+  late CoursesRepository coursesRepository;
+
+  setUp(() {
+    coursesRepository = MockCoursesRepository();
+    coursesCubit = CoursesCubit(coursesRepository);
+  });
+
+  blocTest(
```
Comment (rzhaxibayev) [inline test/features/courses/ui/courses_cubit_test.dart:23]: add more tests
1. how about when coursesRepository.getAllCourses() returns empty list
2. what if coursesRepository.getAllCourses() throws exception?
3. tests for cubit.init() method?

File: test/features/courses/ui/courses_cubit_test.dart [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+              progress: 40,
+              imageUrl:
+                  'https://it-cube48.ru/wp-content/uploads/2020/08/Chto-takoe-Python_1000-0.jpg',
+            ),
+          ]);
+      return coursesCubit;
+    },
+    act: (coursesCubit) async {
+      coursesCubit.init();
+      coursesCubit.searchQuery('Основы программирования(Python)');
```
Comment (rzhaxibayev) [inline test/features/courses/ui/courses_cubit_test.dart]: add test case for searching with different text

File: test/features/courses/ui/courses_cubit_test.dart (hunk 1 above)
Comment (aleke88884) [reply test/features/courses/ui/courses_cubit_test.dart:23]: fixed

File: test/features/courses/ui/courses_cubit_test.dart (hunk 2 above)
Comment (aleke88884) [reply test/features/courses/ui/courses_cubit_test.dart]: fixed

**Review by rzhaxibayev — APPROVED** [review]

**Pull request merged**